packages/cli/src/models/files/ProjectFile.ts

```diff
@@ -58,7 +58,7 @@ export default class ZosPackageFile {
       }.`;
       throw e;
     }
-    checkVersion(this.data.zosversion, this.fileName);
+    checkVersion(this.data.manifestVersion, this.fileName);
```

Comment:
Are we still going to support reading the `zosversion` attribute after these changes for users that have already started their projects with `zos` versions below `2.4.0`?

Reply:
I don't think we should support reading old `zosversion`. I belive we should migrate them to new `manifestVersion`. @spalladino What do you think?

Reply:
Hmm I guess we should support `zosversion` for `zos.json` files, and `manifestVersion` for `.openzeppelin/project.json` files.